src/compiler/index.ts

```diff
@@ -22,4 +22,3 @@ export { path } from './sys/modules/path';
 export { transpile, transpileSync } from './transpile';
 export { version, versions, vermoji, buildId } from '../version';
 export { ts };
-export { InMemoryFileSystem, FsWriteResults } from './sys/in-memory-fs';
```

Comment:
Is `FsWriteResults` re-exported anywhere else? The file that originally had that type definition `stencil-public-compiler` is a signal that this was a publicly exported type - I think we need to continue to re-export it to not break our API (unless I'm missing where it's being re-exported)

Reply:
that makes sense, I will add it back 👍